.gitignore

```diff
@@ -54,3 +54,5 @@ build-iPhoneSimulator/
 
 # Used by RuboCop. Remote config files pulled in from inherit_from directive.
 # .rubocop-https?--*
+
+.idea
```

Comment:
Ruby and Rubocop have a pattern to use the last line of every file how is a blank line, WDYT team, we should follow it or not? :nail_care:

Reply:
it's is applied to .gitignore too ?

Reply:
I don't know, but every file without a last blank line will appear like this:

![image](https://user-images.githubusercontent.com/15777722/152533407-b69c9951-aca1-4fa9-a43e-dd288e1c7f8d.png)

Some projects aren't acetable, my question is about if we will don't care about this or if will put an end line. The VS Code has plugins to do it.

Reply:
I tested here and Rubocop only analyzed Ruby source files, so my answer is for this kind of file I agree to use this pattern, but for the non-ruby files, I think it's not necessary.

WDYT ?

Reply:
It's ok to me 😁